### PR TITLE
Created simple table representing some of Decision model. Resolved #11

### LINKED ISSUE #11: Create simple table representing Decision model

### PR DESCRIPTION
**Pull request by mateuszbochniak**

### CODE REVIEW

**Comment by mariuszkumor** [comment]
@mateuszbochniak could you kindly please attach a screenshot how the table looks like?

**Comment by mateuszbochniak** [comment]
@mariuszkumor  It's a standard Bootstrap table. No special styles added to the lib, or to the testing site... yet.
https://ibb.co/j8pLCz

File: projects/powerflows-testing-app/src/app/app.component.ts [hunk 1]
```diff
@@ -14,13 +14,87 @@
...
+      name: 'Second input (string)',
+      expression: {value: 'r1 for input2', type: ExpressionType.LITERAL}
+    };
+
+    const inputEntry3 = {
+      name: 'Third input (collection)',
+      expression: {value: 'r1 for input3', type: ExpressionType.LITERAL}
+    };
+
+    rule1.inputEntries = [inputEntry1, inputEntry2, inputEntry3];
```
Comment (mariuszkumor) [inline projects/powerflows-testing-app/src/app/app.component.ts:87]: Keep in mind an example having lack of particular input entries (if don't take part in an evaluation process)

**Review by mariuszkumor — DISMISSED** [review]

File: projects/powerflows-testing-app/src/app/app.component.ts (hunk 1 above)
Comment (mateuszbochniak) [reply projects/powerflows-testing-app/src/app/app.component.ts:87]: Yes, I know - it's not the part of the task. Meanwhile, Ive updated styles, it's bit better :)
https://ibb.co/cqQ84e

Comment (mateuszbochniak) [reply projects/powerflows-testing-app/src/app/app.component.ts:87]: I've created new task - check #13

**Comment by mateuszbochniak** [comment]
screen

**Review by mariuszkumor — APPROVED** [review]

**Pull request merged**